src/shared/src/AuditLogApiClient/AuditLogApiClient.ts

```diff
@@ -78,14 +78,17 @@ export default class AuditLogApiClient implements ApiClient {
         httpsAgent,
         headers: {
           "X-API-Key": this.apiKey
-        }
+        },
+        timeout: 5_000
```

Comment:
TIL you can use underscores to make number easier to read!

Reply:
Also though, since this is a shared library it might be safer to pass in the desired timeout (if any) so we don't affect all other uses of it. Maybe pass it in to the constructor and make all calls use it?